source/dashboards/dashboard-cpu-utilization-details-cores.rst

```diff
@@ -1,15 +1,14 @@
 .. _dashboard-cpu-utilization-details-cores:
 
-CPU Utilization Details (Cores)
-================================================================================
-
-.. contents::
```

Comment:
For all but the most complex dashboards, a local table of contents is not needed. They are being removed where possible to declutter the pages.